infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -99,14 +89,26 @@ public LogReplicationConfigManager(CorfuRuntime runtime, String pluginConfigFile
         this.corfuStore = new CorfuStore(runtime);
 
         initStreamNameFetcherPlugin();
-        openTables();
+        openVersionTable();
+        // TODO: delete this method after upgrading
```

Comment:
I would remove this TODO. If you think of it, this is code required for backward/forward compatibility. If you have a client on version A (where that table existed) and he jumps version B (where this table was removed) and directly upgrades to version D you still need to carry on this, due to design changes that were made in version B.